docs/source/gbd2017_models/risk_attributable_causes/diabetes_mellitus/index.rst

```diff
@@ -81,13 +81,17 @@ This cause model is designed to simulate the basic structure of the risk factor
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-1. In vivarium, 'uncomplicated DM Type 1 and Type 2' is 'Moderate', which is different from how GBD 2017 is modelling it. In the future, severity splits will be revisited using disability weights. 
+- In vivarium, 'uncomplicated DM Type 1 and Type 2' is 'Moderate', which is different from how GBD 2017 is modelling it. In the future, severity splits will be revisited using disability weights. 
 
-3. EMR is greater than zero for severe diabetes only.
+- This model has durations for moderate and severe DM that are too long because they arrive to that disease state immediately. Simulants in this model will not progress from moderate to severe.
```

Comment:
The project requires that prevalence for each severity (mild/moderate v. severe) be correct, since this is a condition for treatment of LDL-C.  DM is not a cause of interest for the project beyond its impact on treatment.  We will use prevalence weighted incidence to add new patients to both "moderate" and "severe" states as the simulation runs, and simulants may exit based on the CSMR (detailed below).
These limitations will impact the DM model in a couple ways:
1. The prevalence of each severity should be appropriate, and will be validated against GBD.
2. The rate of mortality from "moderate" will be high, since we are assuming EMR (calculated from CSMR) is the same for both "moderate" and "severe".  It is unclear how this will impact prevalence, since patients in "moderate" will be dying faster (which implies too few simulants in "moderate"), but not transitioning into "severe" (which implies too many simulants in "moderate").
3. This may also impact DM related burden, since progression into more severe states of DM is not possible.  In aggregate, if prevalence is right, the morbidity should be close to GBD estimates, but at the simulant level, we will underestimate burden.